### ISSUE TITLE
Assist with 2 use cases: list all importers and all importees

### ISSUE THREAD

**Issue by fregante** [pahen/madge#309]
I'm constantly looking for the answer to these questions:

- why is file X being bundled?
- what files is file X bundling?

Can Madge help with this somehow effectively? The graph would be good if everything except file X was hidden, because it's impossible to follow the lines otherwise.

**Comment by fregante** [comment]
I just used [Statoscope](http://statoscope.tech/) for this but it requires several clicks to find the dependencies. Example (bottom-right is near the entrypoint, top-left is the direct importer)


<img width="682" alt="Screen Shot 13" src="https://user-images.githubusercontent.com/1402241/149618000-75ab1162-36cd-4ad8-8daf-761963fb2a62.png">

**Comment by PabloLION** [comment]
Good idea. Maybe we can output a rich text like `html` to show all these info.

**Comment by fregante** [comment]
The regular graph should also be ok, what's really missing is a filter that hides everything else.


Given a file X in the middle of the tree, I'd expect a graph that looks like:

```
entry point a -> foo -> FILE X -> bar -> baz
                              -> bur
                              -> can
                              -> boo -> wip 
                                             -> mer
